### PR TITLE
dcd 0.9.0-alpha5 (devel)

### PR DESCRIPTION
**Pull request by John-Colvin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/dcd.rb [hunk 1]
```diff
@@ -16,14 +16,18 @@ class Dcd < Formula
...
-    version "0.9.0-alpha4"
+      :tag => "v0.9.0-alpha5",
+      :revision => "83454f10f49da2c95b366206934cf3a88f8ce3f4"
+    version "0.9.0-alpha5"
   end
 
   depends_on "dmd" => :build
 
   def install
+    if build.stable?
```
Comment (apjanke) [inline Formula/dcd.rb]: This appears to change the `stable` installation by removing the `experimental_allocator` stuff. If that's the case, this PR should probably include a revision bump.

Comment (MikeMcQuaid) [inline Formula/dcd.rb]: Yeh, agreed 👍

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
Yeh, agreed 👍

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again @John-Colvin and for explaining.

**Pull request merged**